Eclecticiq-ic-app/default/data/ui/alerts/create_eclecticiq_sighting.html

```diff
@@ -0,0 +1,58 @@
+<form class="form-horizontal form-complex">
```

Comment:
This file should be in TA. Remove it from here.

Reply:
Okay